sdktests/testapi_hooks.go

```diff
@@ -78,3 +84,28 @@ func (h *Hooks) ExpectCall(t *ldtest.T, hookName string,
 		}
 	}
 }
+
```

Comment:
This is a somewhat janky expectation method. But, we couldn't quite think of what we'd want to see if this was more generic. 

So, I think we should feel free to refactor or replace this when the time comes. Or now, if there's a better idea.